indy_common/types.py

```diff
@@ -68,6 +71,20 @@ class ClientGetNymOperation(MessageValidator):
     schema = (
         (TXN_TYPE, ConstantField(GET_NYM)),
         (TARGET_NYM, IdentifierField()),
+        (TIMESTAMP, IntegerField(optional=True)),
+        (TXN_METADATA_SEQ_NO, TxnSeqNoField(optional=True)),
+    )
+
+
+class ClientNYMOperation(PClientNYMOperation):
+    schema = (
+        (TXN_TYPE, ConstantField(NYM)),
+        (ALIAS, LimitedLengthStringField(max_length=ALIAS_FIELD_LIMIT, optional=True)),
+        (VERKEY, VerkeyField(optional=True, nullable=True)),
+        (TARGET_NYM, DestNymField()),
+        (ROLE, RoleField(optional=True)),
+        (DIDDOC_CONTENT, JsonField(max_length=DIDDOC_CONTENT_SIZE_LIMIT, optional=True)),
```

Comment:
Making `diddocContent` a JSON field makes handling similar to ATTRIB raw data. This makes it trivial to specify a max length. However, it means that the content is double serialized when retrieved with a `GET_NYM` (`diddocContent` is returned as JSON string inside of `data` which is also serialized into a JSON string). I think maintaining consistency is more valuable than optimizing message size but just wanted to call this out.